src/clojure/puppetlabs/ssl_utils/core.clj

```diff
@@ -587,6 +587,16 @@
   (= (.getIssuerX500Principal crl)
      (.getSubjectX500Principal cert)))
 
+(schema/defn ^:always-validate crl-issued-by-cert-in-chain? :- schema/Bool
+  "Given a CRL and a certificate chain, determine whether the CRL was issued by
+  any certificate in the chain."
+  [crl :- X509CRL
+   cert-chain :- [X509Certificate]]
+  (->> cert-chain
+       (filter (partial crl-issued-by-cert? crl))
```

Comment:
could use `some` here instead of needing to have `seq` and `boolean` after filter

Reply:
ahh thanks, I figured there was a better way, but never went back after reworking this. will update.

Reply:
hmm looks like I still need to call `boolean` because `some` returns `nil` if there's no match. I guess I could just remove the output schema, but I'd kinda rather not.. Thoughts?

Reply:
This is a place where clojure idiom kind of clashes with the schema, but yeah I'd go with the schema and keep the `boolean` call.

Reply:
okay cool, updated to use `some` with `boolean`